### PR TITLE
update deprecation policy

### PR DESCRIPTION
**Pull request by vieux**
ping @friism @thaJeztah

### CODE REVIEW

File: engine/index.md [hunk 1]
```diff
@@ -98,8 +98,8 @@ on the separate [Release Notes page](/release-notes)
 As changes are made to Docker there may be times when existing features
 will need to be removed or replaced with newer features. Before an existing
 feature is removed it will be labeled as "deprecated" within the documentation
-and will remain in Docker for, usually, at least 3 releases. After that time
-it may be removed.
+and will remain in Docker for, usually, at least 3 stable releases (roughly 9 months).
```
Comment (mstanleyjones) [inline engine/index.md]: Can we remove `, usually,`? What are the circumstances in which it would be shorter than 3 months?

Comment (vieux) [inline engine/index.md]: @mstanleyjones never shorter, but could be longer.

Comment (mstanleyjones) [inline engine/index.md]: Then "at least" covers it and we don't need "usually".

Comment (vieux) [inline engine/index.md]: I just pushed an update

**Pull request merged**